Misc/NEWS.d/next/Library/2025-01-05-00-55-41.gh-issue-128505.Nf5FY2.rst

```diff
@@ -0,0 +1 @@
+sqlite Connection objects now expose a method set_file_control, which is a thin wrapper for `sqlite3_file_control https://www.sqlite.org/c3ref/file_control.html`_.
```

Comment:
See https://devguide.python.org/documentation/markup/ for help with docs markup.